nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-api/src/main/java/org/nuxeo/ecm/directory/BaseDirectoryDescriptor.java

```diff
@@ -115,11 +148,11 @@
     public Boolean autoincrementIdField;
 
     public boolean isAutoincrementIdField() {
-        return autoincrementIdField == null ? AUTO_INCREMENT_ID_FIELD_DEFAULT : autoincrementIdField.booleanValue();
+        return autoincrementIdField == null ? AUTO_INCREMENT_ID_FIELD_DEFAULT : autoincrementIdField;
```

Comment:
sometimes Intellij give bad advise :(

Reply:
In particular you'll see it a lot in code that I write (also `Boolean.valueOf(somebool)`) because I have warnings on for autoboxing/autounboxing, due to the fact that autounboxing can lead to NPE (and Eclipse has only one setting for autoboxing/autounboxing...).